flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/LogicalSlot.java

```diff
@@ -32,6 +34,20 @@
  */
 public interface LogicalSlot {
 
+	 Payload TERMINATED_PAYLOAD = new Payload() {
+
+		private final CompletableFuture<?> COMPLETED_TERMINATION_FUTURE = CompletableFuture.completedFuture(null);
```

Comment:
nit: `COMPLETED_TERMINATION_FUTURE` should be camel cased because is not actually a constant (not static).